packages/seducer/stories/seducer.stories.mdx

```diff
@@ -0,0 +1,891 @@
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import CollapsibleCard from "@paprika/collapsible-card";
+import Basic from "./examples/useReducer.basic";
+import Blocks, { Block } from "storybook/components/Blocks";
+import BasicWithContext from "./examples/useReducerWithContext.basic";
+import CharacterList from "./examples/useReducerWithContext.CharacterList";
+import ConfettiUseSeducer from "./examples/useReducer.confetti";
+import AsyncExample from "./examples/useSeducer.async";
+
+<Meta title={`${getStoryName("seducer")}/Docs`} />
+
+<!-- <Description markdown={Readme} /> -->
```

Comment:
It would be nice to use the Readme.md file you curated already. Then we only really need to maintain one place instead of two different areas.

This is really for the description and paragraph text you have about the custom hook.

Reply:
done